models/renalStructures_UNEST_segmentation/docs/README.md

```diff
@@ -1,7 +1,9 @@
 # Description
 A pre-trained model for inferencing volumetric (3D) kidney substructures segmentation from contrast-enhanced CT images (Arterial/Portal Venous Phase).
+
 A tutorial and release of model for kidney cortex, medulla and collecting system segmentation.
 
+>>>>>>> db7fd87494d8c7ad9b766bdf6e4010c57a205c79
```

Comment:
Please remove the git conflicts comment.

Thanks.

Reply:
Thanks, I'm not aware of these conflict lines. Checked for this file. 
Thanks again.